guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -4,15 +4,12 @@
 ifdef::foreman-el,katello,satellite[]
 * xref:#storage-el-8[{EL} 8]
```

Comment:
I'd drop this since it's just a single item in a list.

Reply:
Addressed :+1: